src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py

```diff
@@ -783,7 +812,7 @@ def mocked_create_merge_directory(_):
     engine_impl = engine.LibFuzzerEngine()
     options = engine_impl.prepare(corpus_path, target_path, DATA_DIR)
     options.arguments.append('-runs=10')
-    engine_impl.fuzz(target_path, options, TEMP_DIR, 10)
+    engine_impl.fuzz(target_path, options, TEMP_DIR, get_fuzz_timeout(5))
```

Comment:
This get_fuzz_timeout call is confusing, isnt it this signature ?
src/python/bot/fuzzers/libfuzzer.py:def get_fuzz_timeout(is_mutations_run, total_timeout=None):

Reply:
This is a different get_fuzz_timeout, local to this file: https://github.com/google/clusterfuzz/blob/dc099ac6e2eb2e170803a21640acb70b29826580/src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py#L441